From 38acbc6ea62634539b860a152c75c0a9ede95396 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Mon, 27 Jan 2025 13:50:43 -0600
Subject: [PATCH] Revert "mmc: sdhci-omap: Allow SDIO card power off and enable
 aggressive PM"

This reverts commit 3edf588e7fe00e90d1dc7fb9e599861b2c2cf442.

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 drivers/mmc/host/sdhci-omap.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/mmc/host/sdhci-omap.c b/drivers/mmc/host/sdhci-omap.c
index bfb7c8b96341..408f4a6a8154 100644
--- a/drivers/mmc/host/sdhci-omap.c
+++ b/drivers/mmc/host/sdhci-omap.c
@@ -1340,9 +1340,6 @@ static int sdhci_omap_probe(struct platform_device *pdev)
 	/* R1B responses is required to properly manage HW busy detection. */
 	mmc->caps |= MMC_CAP_NEED_RSP_BUSY;
 
-	/* Allow card power off and runtime PM for eMMC/SD card devices */
-	mmc->caps |= MMC_CAP_POWER_OFF_CARD | MMC_CAP_AGGRESSIVE_PM;
-
 	ret = sdhci_setup_host(host);
 	if (ret)
 		goto err_rpm_put;
-- 
2.45.2

